src/main/java/org/spongepowered/api/Game.java

```diff
@@ -22,7 +22,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
```

Comment:
There should be a blank line between the header and package declaration.
Any way to enforce that (e.x in the `HEADER.txt`)?